apps/public-data/src/main.ts

```diff
@@ -0,0 +1,17 @@
+import { NestFactory } from '@nestjs/core';
+import { MicroserviceOptions, Transport } from '@nestjs/microservices';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
+  const PORT = Number(process.env.PUBLIC_DATA_PORT);
+  const app = await NestFactory.createMicroservice<MicroserviceOptions>(AppModule, {
+    transport: Transport.TCP,
+    options: {
+      host: 'localhost',
+      port: PORT,
+    },
+  });
+  await app.listen();
+  console.info(`public-data-service Running On ${PORT} for TCP`);
+}
+bootstrap();
```

Comment:
사소하다고 볼 수 있지만 개행을 의식적으로 해보시는것도 좋을 것 같습니다.
```suggestion
async function bootstrap() {
  const PORT = Number(process.env.PUBLIC_DATA_PORT);
  
  const app = await NestFactory.createMicroservice<MicroserviceOptions>(AppModule, {
    transport: Transport.TCP,
    options: {
      host: 'localhost',
      port: PORT,
    },
  });
  
  await app.listen();
  console.info(`public-data-service Running On ${PORT} for TCP`);
}

bootstrap();
```

Reply:
보기 더 편하네요... 의식해서 적용시켜보겠습니다

Reply:
[5b35177](https://github.com/GrebbMe/grabbme-be/pull/13/commits/5b35177b1ab9350b8e465fdcc1246ec969681e7a)
커밋에 반영하였습니다

Reply:
올려주신 커밋은 해당 리뷰에 대한 작업 커밋이 아닌 것 같습니다.

Reply:
[d6cf5c9](https://github.com/GrebbMe/grabbme-be/pull/13/commits/d6cf5c9502ec90cb5a9a1e3584ae7394e912908b)
멘션을 잘 못 했습니다!
위의 커밋에 반영 하였습니다!